packages/ui/src/components/va-accordion/VaAccordion.stories.ts

```diff
@@ -1,12 +1,167 @@
-import { defineComponent } from 'vue'
-import VaAccordion from './VaAccordion.demo.vue'
+import { VaAccordion } from './'
+import { VaCollapse } from '../va-collapse'
+
+interface Args {
+  modelValue: boolean[]
+  multiple: boolean
+  inset: boolean
+  popout: boolean
+}
+
+const VaAccordionSetup = {
+  value: [false, false, false],
+  valueArray: [false, true, false],
+  collapses: [
+    { title: 'First collapse', content: 'first collapse content' },
+    { title: 'Second collapse', content: 'second collapse content' },
+    { title: 'Third collapse', content: 'third collapse content' },
+  ],
+}
 
 export default {
   title: 'VaAccordion',
   component: VaAccordion,
+  tags: ['autodocs'],
 }
 
-export const Default = defineComponent({
-  components: { VaAccordion },
-  template: '<VaAccordion/>',
+export const Default = (args: Args) => ({
+  components: { VaAccordion, VaCollapse },
+  setup () {
+    return VaAccordionSetup
+  },
+  template: `
+    <va-accordion style="width: 400px;">
+      <va-collapse
+        v-for="(collapse, index) in collapses"
+        :key="index"
+        :header="collapse.title"
+      >
+        <div>
+          {{ collapse.content }}
+        </div>
+      </va-collapse>
+    </va-accordion>
+  `,
+})
+
+export const DefaultWithCustomHeader = (args: Args) => ({
```

Comment:
We can probably remove this demo as it has nothing to do with accordion. (it's collapse specific)